### PR TITLE
:name_badge: Snyk badge and formatting

### PR DESCRIPTION
**Pull request by st3v3nhunt**

### CODE REVIEW

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://profiles-pr-78.dev.c2s.nhschoices.net](http://profiles-pr-78.dev.c2s.nhschoices.net)

File: README.md [hunk 1]
```diff
@@ -20,11 +20,23 @@ the application is being run. This is best practice as described by
...
 
-Start by cloning the repo and all submodules i.e. `git clone https://github.com/nhsuk/profiles.git && cd profiles/ && git submodule update --init --recursive`
-
-It is good practice to run `docker-compose down -v` before starting the application as this will clear up any resources from previous containers that might otherwise affect the correct running of the application.
-Run the application with Docker via `docker-compose up --build --force-recreate profiles-frontend`. This will build an image based on the code in the current working directory and start it running. It will be available locally on [http://localhost:3000](http://localhost:3000)
-
-When finished with the application `docker-compose down -v` should be run to shutdown all services, including volumes. This is the correct way to close down resource used by the `up` command.
-
-It is good practice to run the tests in a Docker container. To do this run `docker-compose -f docker-compose-tests.yml up --build --force-recreate tests`. A new container will be started where the tests will run and rerun when changes are made to the source code.
+Start by cloning the repo and all submodules i.e. `git clone
```
Comment (markysoft) [inline README.md]: I'm not keen on splitting the command text across lines as it breaks the formatting in my editor, and means I can't copy and paste from the readme file, I have to do it from the preview.

Comment (st3v3nhunt) [reply README.md]: Good point. I've put all commands on a single line.

**Comment by c2s-dev** [comment]
:rocket: deployed to [http://profiles-pr-78.dev.c2s.nhschoices.net](http://profiles-pr-78.dev.c2s.nhschoices.net)

**Review by markysoft — APPROVED** [review]
LGTM

**Pull request merged**